docs/examples.rst

```diff
@@ -26,9 +26,9 @@ Basic example
     {
         public function initialize(array $config)
         {
-            $this->table('users');
-            $this->displayField('name');
-            $this->primaryKey('id');
+            $this->setTable('users');
```

Comment:
Mind adding a note here saying these are for 3.4+, and having the older ones commented out for those who have yet to upgrade cake core?